### ISSUE TITLE
Integration to Consul

### ISSUE THREAD

**Issue by linux-lukas** [OpenSlides/OpenSlides#6138]
**Is your feature request related to a problem? Please describe.**
Integration of Free/Libre Open Source Software (FLOSS) into OpenSlides.

**Describe the solution you'd like**
I could well imagine, against the background of OpenData, transparency and trust, that it could make sense for the documents from OpenSlides to be implementable in DKAN. In my view, local parliaments could benefit greatly from this.

**Describe alternatives you've considered**
Manual input.

**Additional context**

reference: https://getdkan.org

**Comment by tsiegleauq** [comment]
As far as I observed: DKAN is a CMS to display files and data sets. I don't really see any meaningful intersections with OpenSlides.

**Comment by Lukas2112** [comment]
Right. Well, my thinking was that if municipalities were to use DKAN for OpenData, and the city council were to use OpenSlides, they could deposit documents (directly) in DKAN?

**Comment by Lukas2112** [comment]
In the meantime, a connection between DKAN and OpenSlides seems to make less sense to me, at least at the moment, since it would also work manually. One idea that seems more sensible to me in this respect would be a connection to Consul.

Example 00: A city administration uses FLOSS Consul for the digital participation of citizens in the case of ideas from the city administration or citizens' ideas. Proposals from citizens and the city administration are submitted and discussed. The city council uses OpenSlides. If there is now an interface, one could look at implementing a function that makes it possible to introduce the citizen's idea to the council. 

@tsiegleauq

**Comment by Lukas2112** [comment]
Besides Consul, could it perhaps also make sense if there were integration in OParl, @tsiegleauq ?

**Comment by tsiegleauq** [comment]
> Besides Consul, could it perhaps also make sense if there were integration in OParl, @tsiegleauq ?

This one https://oparl.org/ ?
The last update was on 25 Jun. 2018, so it seems pretty unmaintained/dead. I suppose OpenSlides already replaces this.

**Comment by Lukas2112** [comment]
Yes, that would be the project. You seem to be right about it seeming unmaintained/dead.

**Comment by Lukas2112** [comment]
How hard would it be to implement an optionally activeable function to transfer a decision made in OpenSlides as a voting into a Consul Project instance (https://consulproject.org/en)?

**Comment by Elblinator** [comment]
Since this is really old I'll close this for now
If this feature is still wanted please open another Issue.
